README.md

```diff
@@ -14,6 +14,21 @@ in Conduit is measured from when it's read from a source until it's ack-ed by al
 the last part is almost 0ms (the NoOp destination is a standalone plugin, so there's a bit of gRPC overhead). This makes
 our measurement of how long a record spends in Conduit pretty accurate.
 
+### Load parameters
+
+The following load parameters are used in tests:
+
+1. The rate of messages generated by a source (expressed in msg/s)
+2. Size of messages generated by a source
+
+### Performance parameters
+
+The performance parameters this tool records can be found in the [`Metrics`](https://github.com/ConduitIO/streaming-benchmarks/blob/b05577ea04932da48f0e620ed912b61d4c42d63b/main.go#L36) struct. The ones we use most when interpreting results are:
```

Comment:
Who needs line breaks 😶‍🌫️

Reply:
I'm not sure I understood.:)

Reply:
I'm saying the line has 245 characters, would be nice to break it up a bit 🙃

Reply:
Ah, I didn't even notice it. I added this quickly through GitHub's web editor, and it appeared as new lines, but now I see that despite it's still the same line number.:)